Autopilot/Relay/AutoRelay_v0_01.c

```diff
@@ -77,6 +86,8 @@ unsigned int timePeriod1 = 0;         // Used in interrupt
 unsigned int timePeriod2 = 0;
 unsigned int t[3] = {0,0,0};                // Used in interrupt
 unsigned int i = 0;
+static unsigned long int lastCaptureTime; //Used in tracking time between edges
```

Comment:
this variable needs to be declared as volatile, since it is set in an interrupt service routine and read outside int he main code